common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -64,13 +87,103 @@ public MojoScorer(
    * @return response {@link ScoreResponse}
    */
   public ScoreResponse score(ScoreRequest request) {
-    MojoFrame requestFrame = requestConverter.apply(request, pipeline.getInputFrameBuilder());
+    MojoFrame requestFrame = scoreRequestConverter
+            .apply(request, pipeline.getInputFrameBuilder());
     MojoFrame responseFrame = doScore(requestFrame);
-    ScoreResponse response = responseConverter.apply(responseFrame, request);
+    ScoreResponse response = scoreResponseConverter.apply(responseFrame, request);
     response.id(pipeline.getUuid());
+    try {
+      ShapleyType requestedShapleyType = shapleyType(request.getShapleyValuesRequested());
+      switch (requestedShapleyType) {
+        case TRANSFORMED:
+          response.setFeatureShapleyContributions(computeContribution(request));
+          break;
+        case ORIGINAL:
+          log.info(UNIMPLEMENTED_MESSAGE);
+          break;
+        default:
+          break;
+      }
+    } catch (Exception e) {
+      log.info("Failed shapley values: {}, due to: {}", request, e.getMessage());
```

Comment:
In general, request can contain sensitive information. It's a bad practice to log content of request as logs can be persisted anywhere.

Reply:
This is from an existing pattern of the score request where the request is logged when there is an exception. I will remove it from here, but will require a refactor of the existing codebase to remove logging of the request